apps/emqx_machine/src/emqx_machine.erl

```diff
@@ -131,7 +131,6 @@ reboot_apps() ->
     , esockd
     , ranch
     , cowboy
-    , ekka
```

Comment:
so there is no need to stop this app and restart ?

Reply:
I moved the restart of ekka to the ekka itself, to make sure it's stopped last and started first. This PR can't be merged without bumping the ekka version.

- [ ] Bump ekka